src/component/sdkKey.ts

```diff
@@ -0,0 +1,40 @@
+import { v } from "convex/values";
+import { internalMutation, internalQuery } from "./_generated/server";
+
+export const store = internalMutation({
+  args: { sdkKey: v.string() },
+  handler: async (ctx, { sdkKey }) => {
+    const existing = await ctx.db.query("sdkKeys").first();
+    if (existing) {
+      throw new Error(
+        'An sdkKey token already exists. Run the "replace" function instead of "store" to update your SDK key.'
+      );
+    }
+    await ctx.db.insert("sdkKeys", {
+      key: sdkKey,
+    });
+    console.log("SDK key stored.");
+  },
+});
+
+export const replace = internalMutation({
+  args: { sdkKey: v.string() },
+  handler: async (ctx, { sdkKey }) => {
+    const existing = await ctx.db.query("sdkKeys").first();
+    if (existing) {
+      await ctx.db.delete(existing._id);
+    }
+    await ctx.db.insert("sdkKeys", {
+      key: sdkKey,
+    });
+    console.log("SDK key stored.");
+  },
+});
```

Comment:
I opted to make a separate function for replacing your SDK key instead of always having it upsert to make it harder to accidentally replace your SDK Key.